src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: circle, area: " + calculateArea() + " sq. units, "

```

Comment:
Use the 'name()' method for getting String representation of enum constants instead of using 'toString()' or 'String.valueOf()' in the 'getColor()' method.